guides/common/modules/ref_dynamic_partition_example.adoc

```diff
@@ -1,8 +1,9 @@
 [id="dynamic-partition-example_{context}"]
 = Dynamic partition example
 
-Using an Anaconda kickstart template, the following section instructs Anaconda to erase the whole disk, automatically partition, enlarge one partition to maximum size, and then proceed to the next sequence of events in the provisioning process:
+Using an Anaconda Kickstart template, the following section instructs Anaconda to erase the whole disk, automatically partition, enlarge one partition to maximum size, and then proceed to the next sequence of events in the provisioning process:
```

Comment:
I know that you were just fixing the capitalization of Kickstart. The whole sentence is weird. Just a comment. No need to change it.